test/unittests/mongoBackend/mongoUpdateContextCompoundValues_test.cpp

```diff
@@ -141,19 +141,21 @@
                                                                          \
     x    = str->add(orion::ValueTypeVector, "x", "");                    \
     leaf = str->add(orion::ValueTypeNumber, "y", 3.0);                   \
+    leaf = str->add(orion::ValueTypeNone, "z", "");                      \
```

Comment:
Fixed in 8414f1791b653f0810868e5d209aecba3732c1e4 + 
ac8acb1a2951419f8e663497a3e22c76a7bfbc90